frontend/e2e/app.po.ts

```diff
@@ -50,11 +50,59 @@ export class TreesSidebarPage {
   getTreeSpecies(type, index) {
     return element(by.id(`tree-${type}-species-${index}`))
   }
+
+  cuttingAreaDates(index) {
```

Comment:
Curious why you are declaring these named functions for just pulling up ids in the e2e?

Reply:
@lauraGgit sorry, missed this one. We'll add a task in a sprint-4 story.

Reply:
task added in this story: https://github.com/nciinc/fs-intake-module/issues/12